Allow importing "cs:" charms also when a model is not upgrading.

This is needed by test code wanting to upload fake charms to fake-juju,
so it doesn't reach out the real charm store.

Index: 2.0.2/src/github.com/juju/juju/apiserver/charms.go
===================================================================
--- 2.0.2.orig/src/github.com/juju/juju/apiserver/charms.go
+++ 2.0.2/src/github.com/juju/juju/apiserver/charms.go
@@ -284,7 +284,9 @@ func (h *charmsHandler) processPost(r *h
 		if isImporting, err := modelIsImporting(st); err != nil {
 			return nil, errors.Trace(err)
 		} else if !isImporting {
-			return nil, errors.New("cs charms may only be uploaded during model migration import")
+			// XXX fake-juju: start patch
+			// return nil, errors.New("cs charms may only be uploaded during model migration import")
+			// XXX fake-juju: end patch
 		}
 
 		// If a revision argument is provided, it takes precedence
